app/controllers/mixins/generic_show_mixin.rb

```diff
@@ -128,7 +148,7 @@ def nested_list(table_name, model, options = {})
       drop_breadcrumb(:name => _("%{name} (All %{title})") % {:name => @record.name, :title => title},
                       :url  => show_link(@record, :display => @display))
 
-      view_options = {:parent => @record}
+      view_options = {:parent => @record, :association => options[:association]}
```

Comment:
https://github.com/martinpovolny/manageiq-ui-classic/blob/master/app/controllers/application_controller.rb#L1504 - nil guard here - that's reason of the comment ```#   association      -- get_view option association - implicit nil``` .. but if you don't like it and rather to use condition ```if options.key?(:association)``` - as the line below, I can use that. please consider that